icu4c/source/configure.ac

```diff
@@ -6,15 +6,16 @@ AC_COPYRIGHT([ Copyright (C) 2016 and later: Unicode, Inc. and others. License &
 
 # NOTE: please use 'autoreconf' to rebuild, otherwise 'aclocal && autoconf'.
 
-# Check for autoconf version
-AC_PREREQ(2.69)
+# Check for minimum autoconf version. Use exactly this version when rebuilding
+# 'aclocal.m4' and 'configure' for merging upstream, to avoid spurious diffs.
+AC_PREREQ([2.72])
 
 # Process this file with autoreconf to produce a configure script
 AC_INIT([ICU],
-        m4_esyscmd_s([sed -n 's/^[ 	]*#[ 	]*define[ 	]*U_ICU_VERSION[ 	]*"\([^"]*\)".*/\1/p' "./common/unicode/uvernum.h]"),
-        [http://icu-project.org/bugs],
-        [International Components for Unicode],
-        [http://icu-project.org])
+        m4_esyscmd_s([sed -n 's/^[ 	]*#[ 	]*define[ 	]*U_ICU_VERSION[ 	]*"\([^"]*\)".*/\1/p' common/unicode/uvernum.h]),
+        [https://icu.unicode.org/bugs],
```

Comment:
it must be 2024!

Reply:
🤣 Updating the URLs wasn't strictly necessary to make autoconf 2.72 accept this without warnings, but I just couldn't resist sneaking that change in …